.github/workflows/build.yml

```diff
@@ -1,11 +1,7 @@
-name: CI
+name: Build
```

Comment:
this is a shared workflow that the other workflows call - includes all the tests/builds etc